api/ooniapi/measurements.py

```diff
@@ -800,6 +807,18 @@ def list_measurements() -> Response:
         query_params["test_name"] = test_name
         fpwhere.append(sql.text("test_name = :test_name"))
 
+    if software_version is not None:
+        query_params["software_version"] = software_version
+        fpwhere.append(sql.text("software_version IN :software_version"))
```

Comment:
We're using `IN` rather than string equality because we want to match patterns--correct?

Reply:
`IN` matches the presence of a version in an array of accepted versions

Reply:
Done